tests/unit/amo/sagas/test_collections.js

```diff
@@ -370,6 +373,50 @@ describe(__filename, () => {
         const action = await sagaTester.waitFor(expectedAction.type);
         expect(action).toEqual(expectedAction);
       });
+
+      it('records the beginning of a modification in state', async () => {
+        const collectionSlug = 'my-slug';
+
+        mockApi
+          .expects('updateCollection')
+          .returns(Promise.resolve());
```

Comment:
Does updating the collection not return any params? Or do we just go on status code to know if the update was successful or not? It seems odd not to care about the API response and not to mock it.

Reply:
In the saga we don't do anything with the response. We assume no error means it worked. This is how the code was when Kumar originally wrote it, so I think it's fine.